README.md

```diff
@@ -2,12 +2,12 @@
 
 RealEstate is a spigot plugin built on top of GriefPrevention to provide players with an ability to sell and rent claims to other players.
 
-The plugin is documented in the [GitHub wiki](https://github.com/EtienneDx/RealEstate/wiki).
```

Comment:
I'd prefer keeping things in the wiki (which you should also be able to edit now), but I'm okay either way